Docs/Book/C++Examples/CMakeLists.txt

```diff
@@ -0,0 +1,37 @@
+cmake_minimum_required( VERSION 2.8.9 )
+
+project(RDKitSV)
+
+set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_SOURCE_DIR}")
+
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -std=c++11" )
```

Comment:
This doesn't look right.  We build on non c++11 systems so enforcing it seems, odd to me.